assets/js/src/core/modules/notifications/send-notification/components/user-select/user-select.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * This source file is available under the terms of the
+ * Pimcore Open Core License (POCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (https://www.pimcore.com)
+ *  @license    Pimcore Open Core License (POCL)
+ */
+
+import { useUserGetCollectionQuery } from '@Pimcore/modules/user/user-api-slice.gen'
+import { Select } from 'antd'
+import React from 'react'
+
+interface UserSelectProps {
+  onChange: (value: string | null) => void
+}
+
+export const UserSelect = ({ onChange }: UserSelectProps): React.JSX.Element => {
```

Comment:
hmm, I'm thinking if we should move that to the user module.
It seems like something we could reuse also in other places where we need a simple user list in a select. Am I right?

Reply:
yeah thats absolutly right. I should we move it to the components folder or to `/modules/user`?

Reply:
moved it to `/modules/user` https://github.com/pimcore/studio-ui-bundle/pull/1515/commits/987bbc4c2fa548a3615ba26a90c6434d3736bcb8